test/suites/ACID.m

```diff
@@ -2434,6 +2435,26 @@
     set(ax4, 'Color', 'none');
 end
 % =========================================================================
+function [stat] = histogramPlot()
+
+  if ~exist('histogram')
```

Comment:
@PeterPablo @okomarov I was confusing `isHG2` with a related function of R2014b that explicitly checks whether an handle graphics object is a HG2/MCOS objects or HG1 one. Sorry for the confusion. I agree with using `isHG2()`.